R/hooks/post_push

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+. ../docker-hooks.sh
+
+image="clearlinux/r-base"
+package=r-base
```

Comment:
The package you are tracking is not "r-base".
There is no "r-base" package in clear linux.
It should be "R".